src/module-vsbridge-indexer-agreement/etc/module.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Divante_VsbridgeIndexerAgreement" setup_version="1.0.0">
+        <sequence>
+            <Divante_VsbridgeIndexerCore/>
```

Comment:
We should also add dependency to Magento core module `Magento_CheckoutAgreements`

Reply:
fixed